src/playground/services/invoices/update/endpoint.clj

```diff
@@ -17,4 +18,8 @@
                    (h/format))
         _      (jdbc/execute! db update)]
 
-    {:status 302 :headers {"Location" "/invoices"} :body "" :flash (str "Entry " id " has been updated.")}))
+
+    (-> (ring-resp/redirect "/invoices")
```

Comment:
I think we found a way to avoid the hardcoded ` "/invoices"`? One should be able to get that string by the route's name (a keyword)